src/lib.rs

```diff
@@ -2,7 +2,6 @@
 // which is licensed under CC0 1.0 Universal
 // https://github.com/davidB/tracing-opentelemetry-instrumentation-sdk/blob/d3609ac2cc699d3a24fbf89754053cc8e938e3bf/LICENSE
 
-use opentelemetry_resource_detectors::OsResourceDetector;
```

Comment:
Wee need `opentelemetry-resource-detectors@0.4.0` here, but it's not released yet

Reply:
Latest available version os `0.3.0`: https://crates.io/crates/opentelemetry-resource-detectors

Reply:
But on the other hand, we don't really need this `OsResourceDetector` in the first place

Reply:
I suggest we just drop `OsResourceDetector` and don't look back, it's not needed at all (we only have linux anyway)

Reply:
Sounds good